molecule/README.md

```diff
@@ -0,0 +1,42 @@
+Molecule tests for the role.
+
+# Test Matrix
+
+Test options "flow down" thro table unless changed.
+
+test   | # partitions | # groups in partitions? | other
+---    | ---          | ---                     | ---
+test1  | 1            | N                       | 2x compute node, sequential names (default test)
+test1b | 1            | N                       | 1x compute node
+test1c | 1            | N                       | 2x compute nodes, nonsequential names
+test2  | 2            | N                       | 4x compute node, sequential names
+test3  | 1            | Y                       |
+
+# Local Installation & Running
+
+Local installation on a Centos7 machine looks like:
+
+    sudo yum install -y gcc python3-pip python3-devel openssl-devel python3-libselinux
+    sudo yum install -y docker-ce docker-ce-cli containerd.io
+    sudo yum install -y yum-utils
+    sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+    sudo yum install -y docker-ce docker-ce-cli containerd.io
+    python3 -m pip install --upgrade --user pip
```

Comment:
we should probably have l24-l29 under molecule/requirements.txt

Reply:
Fixed